src/Monolog/Handler/NativeMailerHandler.php

```diff
@@ -40,6 +40,12 @@ class NativeMailerHandler extends MailHandler
     protected $headers = array();
 
     /**
+     * Optional parameters for the message
+     * @var array
+     */
+    protected $parameters = array();
```

Comment:
It's ok IMO since header is already protected, might as well keep it all open for extension.